src/components/Main.js

```diff
@@ -1,12 +1,13 @@
 import { react, html, css, npm } from '../utils/rplus.js';
 import { useStateValue } from '../utils/globalState.js';
 import { parseUrl } from '../utils/parseUrl.js';
+import AnalyzeFile from '../utils/analyzeFile.js?worker';
 
 import Nav from './Nav.js';
 import Article from './Article.js';
 
 const replaceState = url => history.replaceState(null, null, url);
-const worker = new Worker('./utils/analyzeFile.js');
+const worker = new AnalyzeFile();
```

Comment:
This is pretty dope. Vite supports webworkers out of the box, so we can eventually write this `analyzeFile` file in TS.